assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convertors.py

```diff
@@ -218,30 +219,42 @@ def get_model_signature(self) -> ModelSignature:
                         CLIPMLFlowSchemaLiterals.INPUT_COLUMN_TEXT),
             ]
         )
-        output_schema = Schema(
```

Comment:
Why don't we add a output signature for embeddings task?
From what I understand, having the output signature also would work with returning only image_features or text_features, right?